src/avfoundation.cs

```diff
@@ -8216,6 +8216,37 @@ interface AVCaptureStillImageOutput {
 #endif
 	}
 
+	[NoTV, iOS (10,0), NoMac, NoWatch]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
```

Comment:
To ease *future* review(s) add `// init NS_UNAVAILABLE`
That makes it clear it was Apple intent